Tabloid/Controllers/PostController.cs

```diff
@@ -27,9 +27,14 @@ public IActionResult Get()
 
         // GET api/<PostController>/5
         [HttpGet("{id}")]
-        public string Get(int id)
+        public IActionResult Get(int id)
```

Comment:
Should any authed user be able to see any post? If not we might need to figure out the logic to make sure that this route is protected. For example, if a post isn't published who should be able to see it?

Reply:
This has been changed. If a post is not yet published or is not yet approved, only admins and the post's author may view the details of the post. All users logged in may view details of approved, already published posts.